src/blockMonitor/blockProcessor.ts

```diff
@@ -1,7 +1,8 @@
 import { ethers } from "ethers";
-import { StartStopService } from "../dataEntities";
-import { BlockCache } from "./blockCache";
+import { StartStopService, ApplicationError } from "../dataEntities";
+import { ReadOnlyBlockCache, BlockCache } from "./blockCache";
 import { IBlockStub } from "./blockStub";
+import { waitForEvent } from "../utils";
```

Comment:
Is this used?

Reply:
I ended up not using it, instead. Removed the import.